db/migrate/20150429113903_change_scans.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeScans < ActiveRecord::Migration
+  def change
+    change_table :scans do |t|
+      t.string  :uuid, null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.